src/price_data.py

```diff
@@ -120,7 +120,12 @@ def _get_price_binance(
 
         if len(data) == 0:
             log.warning("Binance offers no price for `%s` at %s", symbol, utc_time)
-            return decimal.Decimal()
+            if quote_asset == "USDT":
+                return decimal.Decimal()
+            log.warning(f"Trying {base_asset}USDT and {quote_asset}USDT")
+            usdt = self.get_price("binance", base_asset, utc_time, "USDT")
+            quote = self.get_price("binance", quote_asset, utc_time, "USDT")
```

Comment:
If `quote_asset : USDT` doesn't exist, this function returns decimal.Decimal(). Which makes usdt/quote results in a `decimal.InvalidOperation: [<class 'decimal.DivisionUndefined'>]`.

We could query `quote` first, and catch the case when `quote_asset : USDT` does not exist.